scapy/contrib/dtp.py

```diff
@@ -50,7 +50,7 @@ def getfield(self, pkt, s):
         return remain,lst
 
     def addfield(self, pkt, s, val):
-        return s+reduce(str.__add__, map(str, val),"")
+        return s + reduce(str.__add__, [str(v) for v in val]), "")
```

Comment:
You can remove the list and the useless `reduce()`: `return s + ''.join(str(v) for v in val)`.